chatlearn/synchronizer/parameter_sync.py

```diff
@@ -636,7 +663,6 @@ def sync_broadcast(self, actors, group_name, requires_grad=None, filter_fn=None,
 
     def sync_allgather(self, actors, group_name, requires_grad=None, filter_fn=None):
         for actor in actors:
-            # Currently, only routed experts are to be all-gathered.
```

Comment:
What's the reason for deleting the comment?

Reply:
it went lost when rebasing last commit. Recover the comment already.